utam-compiler/src/test/java/utam/compiler/grammar/DeserializerUtilities.java

```diff
@@ -37,7 +37,7 @@ public DeserializerUtilities() {
     this.translatorConfig = getDefaultConfig();
   }
 
-  private static String readJSON(String fileName) {
+  public static String readJSON(String fileName) {
```

Comment:
why public? it's not used outside this class from what i can see

Reply:
Good catch. Fixed in next commit.